docs/advanced_topics/icons.md

```diff
@@ -80,3 +72,12 @@ Place your app before any Wagtail apps in `INSTALLED_APPS`.
 
 Wagtail icons live in `wagtail/admin/templates/wagtailadmin/icons/`.
 Place your own SVG files in `<your_app>/templates/wagtailadmin/icons/`.
+
+## Available icons
+
+Enable the [styleguide](styleguide) to view the available icons and their names for any given project.
+
+Here are all available icons out of the box:
+
+```{include} ../_static/wagtail_icons_table.txt
+```
```

Comment:
I’ve moved this section to the end of the document so the list of icons doesn’t get in the way of reading other documentation sections.

The included file is `.txt` because we don’t want `djhtml` to run on it. The file’s contents will be processed as _Markdown_, and we also don’t want Prettier to run on it. It’s crucial for correct formatting that there are no spaces at the start of lines (interpreted as code blocks).

Reply:
Just wondering, would it be worth wrapping the table in a `<details open>` component? (note the default `open` attribute)

Reply:
👍 I’ll give that a go.